docs/tutorial/part-zero/index.md

```diff
@@ -10,47 +10,44 @@ Before you start building your first Gatsby site, you’ll need to familiarize y
+### Install Xcode Command Line Tools:
 
 1. Open your Terminal.
-1. On a Mac, install Xcode Command line tools by running `xcode-select --install`.
+1. Install Xcode Command line tools by running `xcode-select --install`.
    1. If that fails, download it [directly from the Apple site](https://developer.apple.com/download/more/), after signing-in with Apple developer account
 1. After being prompted to start the installation, you'll be prompted again to accept a software license for the tools to download.
 
-## ⌚ Install Node.js and npm
+## Installing Node.js and npm
 
-Node.js is an environment that can run JavaScript code outside of a web browser. Gatsby is built with Node.js. To get up and running with Gatsby, you’ll need to have a recent version installed on your computer.
+Node.js is an environment that can run JavaScript code outside of a web browser. Gatsby is built with Node.js. To get up and running with Gatsby, you’ll need to have a recent version installed on your computer. npm comes bundled with Node.js so if you don't have npm, chances are that you don't have Node.js too.
 
 _Note: Gatsby's minimum supported Node.js version is Node 8, but feel free to use a more recent version._
 
 1. Open your Terminal.
-1. Run `brew update` to make sure you have the latest version of Homebrew.
-1. Run this command to install Node and npm in one go: `brew install node`
-
-Once you have followed the installation steps, make sure everything was installed properly:
-
-### Check your Node.js installation
-
-1.  Open up your terminal.
-2.  Run `node --version`. (If you’re new to the command line, “run `command`” means “type `node --version` in the command prompt, and hit the Enter key”. From here on, this is what we mean by “run `command`”).
-3.  Run `npm --version`.
-
-The output of each of those commands should be a version number. Your versions may not be the same as those shown below! If entering those commands doesn’t show you a version number, go back and make sure you have installed Node.js.
-
-![Check node and npm versions in terminal](01-node-npm-versions.png)
+1. Install Node.js for your appropriate operating system.
+   - On **MacOS** with homebrew installed, run `brew install node`. If you don't want to install it through homebrew, download the latest Node.js version from [the official Node.js website](https://nodejs.org/en/), double click on the downloaded file and go through the installation process.
+   - On **Windows**, download and install the latest Node.js version from [the official Node.js website](https://nodejs.org/en/)
+   - On **Linux**, run `sudo apt-get install curl`. After it finishes installing, run `curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -` and install Node.js by running `sudo apt-get install nodejs`. If it asks for confirmation, type `y` and press enter. If you're on Fedora, follow [this guide to install Node.js and npm](https://tecadmin.net/install-latest-nodejs-on-fedora/) instead.
```

Comment:
Hey, I wonder if we should split instructions for different distro earlier, as initial instructions assume debian based (`apt-get`) distro.

Something like:
```md
- On **Linux**:
  - Debian based distributions (Ubuntu, Debian, Pop!_OS, ...):
    <instructions here>
  - Fedora (Fedora, CentOS, ...):
    <instructions here>
  - Arch (Arch, Manjaro, ...):
    <instructions here>
  - Something else - please find instructions on the web (we only list most popular distros here)  
```

I'm also not very deep into distro differences/what's based on what, so my snippet above might be wrong ;) Just food for thought

Reply:
I'm not sure how good people know "linux" but we could split on package manager level?
```md
- On **Linux**:
  - Using apt-get (Ubuntu, Debian, Pop!_OS, ...):
    <instructions here>
  - Using yum (Fedora, CentOS, ...):
    <instructions here>
  - Using pacman (Arch, Manjaro, ...):
    <instructions here>
  - Something else - please find instructions on the web (we only list most popular distros here)  
```

Reply:
I like the idea of splitting by package manager. If someone uses a less popular version of linux and we don't mention it, they might get confused, but they might have installed a package before so they will recognize the command (apt-get or pacman). Also there are way fewer package managers than Linux distros, so if someone doesn't know what they use, they can try each part easier.